browser/ui/views/overlay/brave_video_overlay_window_views.cc

```diff
@@ -341,6 +367,36 @@ void BraveVideoOverlayWindowViews::Hide() {
   VideoOverlayWindowViews::Hide();
 }
 
+int BraveVideoOverlayWindowViews::GetNonClientComponent(
+    const gfx::Point& point) {
+  int res = VideoOverlayWindowViews::GetNonClientComponent(point);
+  if (seeker_ && seeker_->GetWidget() == this) {
+    gfx::Point point_in_seeker(point);
+    views::View::ConvertPointFromWidget(seeker_.get(), &point_in_seeker);
+    if (seeker_->HitTestPoint(point_in_seeker)) {
+      // We want to handel mouse event on seeker when it's visible, rather than
+      // consider it as non-client area
+      return HTCLIENT;
+    }
+  }
+
+  return res;
```

Comment:
nit:
```suggestion
  if (seeker_ && seeker_->GetWidget() == this) {
    gfx::Point point_in_seeker(point);
    views::View::ConvertPointFromWidget(seeker_.get(), &point_in_seeker);
    if (seeker_->HitTestPoint(point_in_seeker)) {
      // We want to handel mouse event on seeker when it's visible, rather than
      // consider it as non-client area
      return HTCLIENT;
    }
  }

  return VideoOverlayWindowViews::GetNonClientComponent(point);;
```

Reply:
Nice!